Exercise2.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Linq1
+{
+    public static class Exercise2
+    {
+        // Given a sequence of words, get rid of any that don't have the character 'e' in them,
+        // then sort the remaining words alphabetically, then return the following phrase using
+        // only the final word in the resulting sequence:
+        //    -> "The last word is <word>"
+        // If there are no words with the character 'e' in them, then return null.
+        //
+        // TRY to do it all using only LINQ statements. No loops or if statements.
+        public static string GetTheLastWord(this IEnumerable<string> words)
+        {
+            // using Lambda expression Syntax
+            // return $"The last word is {words.Where(st => st.Contains("e")).OrderBy(st => st).Select(st => st).ToArray().LastOrDefault()}";
+            string g = words.Where(st => st.Contains("e")).OrderBy(st => st).Select(st => st).ToArray().LastOrDefault();
+            if (g == null)
```

Comment:
you could use a trinary operator like this
return g  != null ? "The last word is {g}" : null;

Reply:
i used this way  frequently ... but i mean can 
How can we use the condition statement inside lambda expression (Linq) without using separte string (g) ? 
Example ==> return .....;

In general, I realized that I was a lazy person 😂😂 , since (g) not mean anything

Reply:
you can do this .Select(st => $"the last world is {st}").FirstOrDefault() ?? "null";